common/flavorization/wc_title_holders.txt

```diff
@@ -42,8 +42,7 @@ warchief_female = {
 
```

Comment:
Resolve comments. The PR is so spammed right now.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113167259-9a62b380-9254-11eb-8fc0-f2761ed6310b.png)

</details>